jest.config.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
   testMatch: ["**/*.spec.ts?(x)"],
   moduleNameMapper: {
     "\\.(jpe?g|png)$": "jest-static-stubs/$1",
-    "\\.s?css$": "identity-obj-proxy",
+    "\\.s?css$(?<!((packages|@kaizen)/design-tokens/(sass|less|css)/.*))":
+      "identity-obj-proxy",
```

Comment:
I've got to admit I don't understand what this is doing. Can this be split out into multiple simpler rules, or can you add a comment explaining it?

Reply:
This was originally saying: "ignore `@kaizen/design-tokens/(sass|less|css)` when mocking SCSS imports in jest"
Originally, if you tried to `require.resolve("@kaizen/design-tokens/sass/color.scss")` you would get `identity-obj-proxy` when executing in Jest, but we wanted the actual SCSS.

I just refactored the code that required this to not require the SCSS files and use the JSON tokens instead, so i reverted it.